core/capabilities/launcher.go

```diff
@@ -225,7 +289,8 @@ func (w *launcher) Launch(ctx context.Context, state *registrysyncer.LocalRegist
 
 	// Now, if my node is a workflow DON, let's setup any shims
 	// to external capabilities.
-	if len(myWorkflowDONs) > 0 {
+	belongsToAWorkflowDON := len(myWorkflowDONs) > 0
```

Comment:
nit: I'm for expressive variable names, but seems redundant with the comment also explaining

Reply:
The longer name was my earlier suggestion :) But yeah maybe the comment is now redundant.